packages/website/src/css/prism.css

```diff
@@ -56,10 +56,10 @@ html[data-theme='dark'] {
   --token-color-highlight: rgba(247, 235, 198, 0.2);
 }
 
-div[class*='codeBlockContainer'],
-div[class*='codeBlockContainer'] pre {
-  color: var(--token-color);
-  background-color: var(--token-background);
+.theme-code-block,
+.theme-code-block pre {
+  color: var(--token-color) !important;
```

Comment:
This is to work around a Docusaurus bug: https://github.com/facebook/docusaurus/issues/3678